ui/pages/confirmations/confirmation/confirmation.js

```diff
@@ -466,10 +466,22 @@ export default function ConfirmationPage({
       ? handleSubmit
       : null);
 
+  let contentMargin = 0;
+  if (pendingConfirmations.length > 1) {
+    contentMargin += 32;
```

Comment:
Minor, would these magic numbers benefit from constants such as `NAVIGATION_CONTROLS_HEIGHT` and `SNAP_DIALOG_EXTRA_HEIGHT`?

Reply:
Yea, good point. I have incoming PR after this one where I will need to update these numbers anyway, so will add the constants there.